### ISSUE TITLE
When will Keen-UI get next update?

### ISSUE THREAD

**Issue by EmilMoe** [JosephusPaye/Keen-UI#404]
I'm curious for when to expect an update next time?

**Comment by JosephusPaye** [comment]
In the next month or two (June-July). It'll be a major release, probably 2.0.

**Comment by EmilMoe** [comment]
Cool. How is the computability with the current latest version?

**Comment by HawkeyePierce89** [comment]
I have a question. In next major release a problem with big build size will be solved? I do build the app using webpack and I want normal tree shaking

![image](https://user-images.githubusercontent.com/25603805/40278250-f52b89e4-5c57-11e8-955d-159de8411cde.png)

**Comment by EmilMoe** [comment]
@JosephusPaye  how is it looking for version 2?

**Comment by JosephusPaye** [comment]
Coming soon, folks. Sorry for the wait.

I plan to:

- Release `v1.1.0` with stability improvements and bug fixes and a few necessary breaking changes. Work on this is mostly done (see https://github.com/JosephusPaye/Keen-UI/commits/next)
	- Changed from Tether to Popper for the dropdowns and tooltips
	- Rewrote UiTabs, fixes a lot of issues
	- Rewrote UiRippleInk
	- Refactor the way we handle props that accept DOM and component refs
	- Refactor focus containment, as used in the modal, select, etc. This allows nested modals and dropdowns
	- Rewrote the customization guide: https://github.com/JosephusPaye/Keen-UI/blob/next/Customization.md
	- More performance improvements and bug fixes. See the [`next` changelog](https://github.com/JosephusPaye/Keen-UI/blob/next/CHANGELOG.md)
	- Release date: aiming for mid to late December, 2018

- Release `v2.0.0`:
	- This will be a deeper refactor to fix some fundamental decisions made early that are leading to a lot of issues related to stability, customization and performance.
	- Release date: aiming for mid to late January, 2018.
	- Update docs site, with inline example snippets, search, better SEO for reaching specific doc pages from search engines like Google.

If you want to help, I'd love some feedback on the changes in next.

To try it:

```sh
# npm
npm install keen-ui@JosephusPaye/Keen-UI#next

# yarn
yarn add keen-ui@JosephusPaye/Keen-UI#next
```

See the [changelog](https://github.com/JosephusPaye/Keen-UI/blob/next/CHANGELOG.md) for all changes.

Please report any issues you find as comments to this thread.

**Comment by EmilMoe** [comment]
I assume you mean *2019* @JosephusPaye  :-)

> Release date: aiming for mid to late January, 2018.

**Comment by JosephusPaye** [comment]
Just released [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
